README.md

```diff
@@ -1,3 +1,50 @@
+# CFDE ICC Evaluation Core
+
+## Overview
+
+The [Common Fund Data Ecosystem (CFDE)](https://commonfund.nih.gov/dataecosystem) is an effort to bring together knowledge across [Common Fund](https://commonfund.nih.gov/) programs into a cohesive resource.
+The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, evaluating the activities of Common Fund programs.
+
+This repository is a place to:
+
+- View a high-level evaluation of Common Fund program activity (e.g. projects over time, dollars awarded, # of publications produced, etc.).
+- Submit details about your Common Fund project to be included in the above.
+- Maintain code used to coordinate the above.
+
+The evaluation can be viewed as a live webapp dashboard or as separate PDF reports.
```

Comment:
```suggestion
Reports are availabe as a live webapp dashboard or as PDFs that will be able to include narrative text and additional analyses.
```

Reply:
Throughout the docs and code comments, I'm trying to define "reports" as the printed pdfs and "pages" as the live dashboard pages. If you'd like me to use "reports" to mean either "pdf reports" or "webapp/page reports", that's okay but I'll want to change all instances of this throughout the repo for consistency (and avoid my own confusion).

Reply:
A change of that scale was not intended. :)

Reply:
It won't take that long if you think trying to make those terms stick will be an uphill battle. Just want to be consistent either way.